datadog/resource_datadog_dashboard.go

```diff
@@ -2946,6 +2946,10 @@ func buildTerraformScatterplotRequest(datadogScatterplotRequest *datadog.Scatter
 
 func getTimeseriesDefinitionSchema() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
+		"type": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
```

Comment:
The `type` is implied by using `timeseries_definition` block so no need to explicitly add it again in the config.